qualification/__init__.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, cbf: CBFRemoteControl, stream_names: Sequence[str]) -> None:
         self.sync_time = cbf.sensors[f"{acv_name}.sync-time"].value
         self.scale_factor_timestamp = cbf.sensors[f"{acv_name}.scale-factor-timestamp"].value
         self.bandwidth = cbf.sensors[f"{acv_name}.bandwidth"].value
-        self.center_freq = cbf.sensors[f"{acv_name}.center-freq"].value
```

Comment:
Are we really changing CBF sensor names?

Reply:
If you prefer, Ludwig, we can add the change to katdal and you can just approve it.
We just had an inconsistency with how it was used and there's no way to make it consistent without breaking *something*.

Reply:
Nah, it's fine, I'll do that part.

It's funny. In my world, consistency means CBF says "center" and the rest of MeerKAT says "centre". I'm used to that by now. :-P

Reply:
I'm against updating these sensor names. katdal is just one of many consumers of sensors, and forcing them all to cope with two possible spellings makes the system less maintainable rather than more.

Reply:
The product-configure schema uses "centre" so either some inconsistency or some level of breakage is unavoidable at this point.

Reply:
That was kind of my point... We've been living with SPEAD / sensor inconsistency for so long that it's turned into consistency for me. Make sure you understand the breakage before you put consistency first though. From my side it is OK to add a workaround to katdal but I'm sympathetic to Bruce's concerns.